test/Modular Design/Encounters.js

```diff
@@ -0,0 +1,94 @@
+class Encounters{
+    static REPORTING_FACILITY_ORG_ID = null;
```

Comment:
is there a reason behind having different variable names casing ? it is best to have variable names that are consistent, I think best upper case maybe all of them ?